src/amenities/amenities.controller.ts

```diff
@@ -91,12 +91,12 @@ export class AmenitiesController {
     description: 'Internal Server Error',
   })
   @ApiBody({ type: AmenitiesDto })
-  addAmenities(@Param('id') id: string, @Body() dto: AmenitiesDto) {
-    return this.amenitiesService.addAmenities(id, dto);
+  addAmenities(@Param('id') id: string, @Body() dto: AmenitiesDto, @Req() req: any) {
+    return this.amenitiesService.addAmenities(id, dto, req.user.id);
   }
```

Comment:
There is a CurrentUser decorator to get user id:

`@CurrentUser('id') userId: string`

Reply:
thanks, changed it